File: java/src/main/java/com/android/widget/FloatWindow/NeedPermission/AATest/TestFloatPageTwoActivity.java
package com.android.widget.FloatWindow.NeedPermission.AATest;

import android.os.Bundle;

import com.android.frame.mvc.BaseActivity;
import com.android.java.databinding.ActivityCommonLayoutBinding;
import com.android.util.CommonLayoutUtil;

/**
 * Created by xuzhb on 2021/3/10
 * Desc:
 */
public class TestFloatPageTwoActivity extends BaseActivity<ActivityCommonLayoutBinding> {

    @Override
    public void handleView(Bundle savedInstanceState) {
        CommonLayoutUtil.initCommonLayout(this, "悬浮窗页面二", false, true, "跳转到悬浮窗页面三");
        binding.tv.setText("悬浮窗A不在页面二显示");
    }

    @Override
    public void initListener() {
        binding.btn1.setOnClickListener(v -> {
            startActivity(TestFloatPageThreeActivity.class);
        });
    }

}

File: kotlin/src/main/java/com/android/widget/FloatWindow/NeedPermission/AATest/TestFloatPageTwoActivity.kt
package com.android.widget.FloatWindow.NeedPermission.AATest

import android.os.Bundle
import com.android.basicproject.databinding.ActivityCommonLayoutBinding
import com.android.frame.mvc.BaseActivity
import com.android.util.initCommonLayout

/**
 * Created by xuzhb on 2021/3/30
 * Desc:
 */
class TestFloatPageTwoActivity : BaseActivity<ActivityCommonLayoutBinding>() {

    override fun handleView(savedInstanceState: Bundle?) {
        initCommonLayout(this, "悬浮窗页面二", false, true, "跳转到悬浮窗页面三")
        binding.tv.text = "悬浮窗A不在页面二显示"
    }

    override fun initListener() {
        binding.btn1.setOnClickListener {
            startActivity(TestFloatPageThreeActivity::class.java)
        }
    }

    override fun getViewBinding() = ActivityCommonLayoutBinding.inflate(layoutInflater)

}